### PR TITLE
Snaplistview with different header height

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] N/A
[Problem] Items on SnapListview don't centered.
[Solution]
Scrolling tool need more changes for using snapSize and snapPoints in the same time.
+ snapSzie & snapPoints work in the same time
+ support for bigger & smaller header, content and footer
+ issue with maxScrollPosition - value of maxScrollPosition is too low in specific cases
+ fix for snap to top of header and bottom of content

### CODE REVIEW

**Comment by TomaszLukawskiSam** [comment]
Swipe list examples:
https://github.com/TomaszLukawskiSam/TAU/tree/snaplistview-examples

**Comment by TomaszLukawskiSam** [comment]
The latest version fixes all issues with items are not positioning to center of screen. This version has support for different height of header and content.
Several issue still exists. Work in progress.

File: src/js/profile/wearable/widget/wearable/Selector.js [hunk 1]
```diff
@@ -334,7 +334,7 @@
 					ITEM_RADIUS: -1,
 					ITEM_START_DEGREE: 30,
 					ITEM_END_DEGREE: 330,
-					ITEM_NORMAL_SCALE: "scale(" + STATIC.SCALE_FACTOR + ")",
+					ITEM_NORMAL_SCALE: "scale(" + STATIC.SCALE_FACTOR.toString() + ")",
```
Comment (pkaczmarczy) [inline src/js/profile/wearable/widget/wearable/Selector.js:337]: I don't think this file should be included

**Review by pkaczmarczy — CHANGES_REQUESTED** [review]

**Comment by pkaczmarczy** [comment]
Please also remove changes from dist directory from this commit.

File: src/js/profile/wearable/widget/wearable/Selector.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/Selector.js:337]: Thanks. This is change from another commit. Will be remove.

**Comment by TomaszLukawskiSam** [comment]
> Please also remove changes from dist directory from this commit.

I agree with you. This will be remove.

**Comment by pkaczmarczy** [comment]
I have reviewed rest of the changes and it looks good to me.
I have also checked attached samples.

**Comment by lslachciak** [comment]
Because currently we are not considering this feature, could you close this PR. We will reopen it when needed.

**Comment by TomaszLukawskiSam** [comment]
At the moment PR is frozen.

**Pull request closed without merging**